src/ray/gcs/gcs_server/gcs_one_event_converter.cc

```
@@ -42,8 +42,18 @@ void GcsOneEventConverter::ConvertToTaskEventData(
       ConvertTaskExecutionEventToTaskEvent(event.task_execution_event(), task_event);
       break;
     }
+    case rpc::events::RayEvent::ACTOR_TASK_DEFINITION_EVENT: {
+      ConvertActorTaskDefinitionEventToTaskEvent(event.actor_task_definition_event(),
+                                                 task_event);
+      break;
+    }
+    case rpc::events::RayEvent::ACTOR_TASK_EXECUTION_EVENT: {
+      ConvertActorTaskExecutionEventToTaskEvent(event.actor_task_execution_event(),
+                                                task_event);
+      break;
+    }
     default:
-      // TODO(can-anyscale): Handle other event types
+      RAY_CHECK(false) << "Unexpected one event type: " << event.event_type();
```

should we throw an exception if this case is hit? logging may swallow the exception causing unintended behaviours in down stream consumers (`GcsTaskManager::HandleAddEvent`)

---

good question; `RAY_CHECK` is not logging, it actually "throws" and terminates ray program